modin/core/execution/dask/implementations/pandas_on_dask/io/io.py

```diff
@@ -89,10 +90,20 @@ def __make_write(*classes, build_args=build_args):
     read_csv_glob = __make_read(
         ExperimentalPandasCSVGlobParser, ExperimentalCSVGlobDispatcher
     )
+    read_parquet_glob = __make_read(
+        ExperimentalPandasParquetParser, ExperimentalGlobDispatcher
+    )
+    to_parquet_glob = __make_write(
+        ExperimentalGlobDispatcher,
+        build_args={**build_args, "base_write": BaseIO.to_parquet},
+    )
     read_pickle_distributed = __make_read(
-        ExperimentalPandasPickleParser, ExperimentalPickleDispatcher
+        ExperimentalPandasPickleParser, ExperimentalGlobDispatcher
+    )
+    to_pickle_distributed = __make_write(
```

Comment:
Should we change `read_pickle_distributed` and `to_pickle_distributed` to `read_pickle_glob` and `to_pickle_glob` (separate issue)?

Reply:
I'd like it for consistency.

Reply:
File an issue for that please.

Reply:
https://github.com/modin-project/modin/issues/6856